java-util/src/main/java/io/druid/java/util/common/StringUtils.java

```diff
@@ -77,11 +78,31 @@ public static String fromUtf8(final byte[] bytes)
     }
   }
 
+  @Nullable
+  public static String fromUtf8Nullable(@Nullable final byte[] bytes)
+  {
+    if (bytes == null) {
+      return null;
+    }
+    return fromUtf8(bytes);
+  }
+
   public static String fromUtf8(final ByteBuffer buffer, final int numBytes)
   {
     final byte[] bytes = new byte[numBytes];
     buffer.get(bytes);
-    return StringUtils.fromUtf8(bytes);
+    return fromUtf8(bytes);
+  }
+
+  @Nullable
+  public static String fromUtf8Nullable(final ByteBuffer buffer, final int numBytes)
+  {
+    if (numBytes < 0) {
```

Comment:
Please add javadoc to the method specifying this.

Reply:
added.